deriv.com.conf

```diff
@@ -1,7 +1,15 @@
+map $status $cache_header {
+    default "public, max-age=7200, s-maxage=600, must-revalidate";
```

Comment:
Note for me, and maybe others:
`must-revalidate` doesn't mean what I thought it did
Two resources i'm reading to understand this better:
* [mdn documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)
* [this post](https://sookocheff.com/post/api/effective-caching/)